static/monorail-2.0.yaml

```diff
@@ -2321,7 +2321,115 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
-
+  /nodes/{identifier}/relations:
+    x-swagger-router-controller: nodes
+    get:
+      operationId: nodesGetRelations
+      x-privileges: [ 'Read' ]
+      x-authentication-type: [ 'jwt' ]
+      summary: |
+        Get a node's relations
+      description: |
+        Get the relations field of a node
+      parameters:
+        - name: identifier
+          in: path
+          description: |
+              The node identifier
+          required: true
+          type: string
+      tags: [ "/api/2.0" ]
+      responses:
+        200:
+          description: Successfully retrieved the node's reltions
+          schema:
+            type: array
+            items:
+              type: object
+        404:
+          description: The node with the identifier was not found.
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    delete:
+      x-swagger-schema: relations.2.0.json#/definitions/Relation
+      operationId: nodesDelRelations
+      x-privileges: [ 'Write' ]
+      x-authentication-type: [ 'jwt' ]
+      summary: |
+        Remove relations from nodes
+      description: |
+        Edit the relations fields of a nodes to remove specific relationships
+      parameters:
+        - name: identifier
+          in: path
+          description: |
+                The node identifier
+          required: true
+          type: string
+        - name: content
+          in: body
+          description: |
+              A json object with relation types as keys and arrays of node ids 
+              as values. The node's given relation types will be updated to remove
+              the nodes given by id 
+          required: true
+          schema:
+            $ref: '#/definitions/generic_obj'
+      tags: [ "/api/2.0" ]
+      responses:
+        204:
+          description: Successfully removed the relations 
+        404:
+          description: The node with the identifier was not found.
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    patch:
```

Comment:
Since this is to add relations, so use POST instead of PATCH. If doesn't support duplicated relation, then use PUT.

Maybe you think in the view point of node, this is to PATCH node information, but since you have use a new resource name `/nodes/{identifier}/relations`, so the HTTP verb need works on the `relations`.